Detections/Corelight/CorelightTypoSquattingOrPunycodePhishingHTTPRequest.yaml

```diff
@@ -0,0 +1,29 @@
+id: 6b579e98-abc9-4e7a-9efc-2f3408ba16c9
+name: Corelight - Possible Typo Squatting or Punycode Phishing HTTP Request
+description: |
+  'Detects when an HTTP request was made to a domain that was using unicode/punycode.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated